otx/mpa/modules/datasets/cls_dir_dataset.py

```diff
@@ -186,6 +198,9 @@ def evaluate(self, results, metric="accuracy", metric_options=None, logger=None)
             self.class_acc = True
 
         eval_results = super().evaluate(results, metrics, metric_options, logger)
+        for k in metric_options["topk"]:
+            eval_results[f"accuracy_top-{k}"] /= 100
```

Comment:
How about 
`eval_results = {key:val/100 if key.startswith('accuracy_top-') else {key:val} for key, val in eval_results.items()}`?

Reply:
@JihwanEom we were discussing the future otx code guideline recently (one of the candidates is https://google.github.io/styleguide/pyguide.html#power-features), lets avoid hard-to-read dict comprehensions with conditions.

Reply:
Okay, thank you for kind explanation!